net.mancubus.SLADE3.appdata.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>net.mancubus.SLADE3</id>
+	<name>SLADE3</name>
+	<summary>It's a Doom editor</summary>
+	<metadata_license>CC0-1.0</metadata_license>
```

Comment:
Is llicensing the matadata under CC0 ok? Citing the AppData spec:
"A permissive [metadata-]license ensures your data can be combined with arbitrary other data in one file, without license conflics (this means copyleft licenses like the GPL are not suitable as metadata license). Possible license identifiers include:

    FSFAP
    CC0-1.0
    CC-BY-3.0
    CC-BY-SA-3.0
    GFDL-1.3
    MIT"